README.md

```diff
@@ -44,7 +44,7 @@ to install Babel 6, they could now do
 
 
 ```sh
-npm i some-package babel-core@^7.0.0-0 @babel/core
+npm i some-package babel-core@^7.0.0-bridge.0 @babel/core
```

Comment:
I am positive that this needs to contain `bridge` though, as I was getting prompted for a version when using `-0` and was getting errors when choosing `beta`.